collections/Exeter_College/Exeter_College_MS_38.xml

```diff
@@ -101,7 +101,7 @@
                   </handDesc>
                   <decoDesc>
                      <decoNote type="flourInit">Four-line blue and red lombards, flourished blue and red, at the beginning of each psalm, some with extenders. Also similar ten- and six-line lombards at the beginnings of the commentary on the ferial psalms, 26 and 38, and plain one-line blue and red initials. Lemmata underlined red. One red line-filler; rubrics. Fragments of acanthus leaves drawn in margins probably indicate points of interest, as may occasional heads. Sometimes a foot stands on the top of an acanthus leaf (fols. 33<hi rend="superscript">v</hi>, 65<hi rend="superscript">v</hi>, 71<hi rend="superscript">v</hi>) and there are occasional hands, not pointing but open as if saying ‘behold!’ (fols. 34<hi rend="superscript">v</hi>, 36<hi rend="superscript">r</hi>). </decoNote>
-                     <decoNote type="miniatures">Snakes are common: on fol. 38<hi rend="superscript">v</hi> a snake of 63 mm length is vertical in the margin and at one point is represented as disappearing into and emerging from the membrane; an elegant 45 mm snake is on fol. 43<hi rend="superscript">v</hi>; there are also a bird’s head, animal heads, and human feet. In the bottom margin of fol. 86<hi rend="superscript">v</hi> is a pencil drawing of a group of buildings, including a batdemented wall or tower, a domed tower, and a building with a red roof. Alexander and Temple, no. 150.</decoNote>
```

Comment:
these sound like they should be classified as "drawing" rather than "miniature" (this is a new distinction in the guidelines- sorry)